README.rst

```diff
@@ -1,4 +1,44 @@
-|PyPI| |Conda| |Cite| |CI| |Docs| |Coverage|
+|PyPI| |Conda| |Cite| |CI| |Docs| |Coverage| |License| |GitHubDownloads| |PyPIdownloads| |CondaDownloads|
+
+.. |PyPI| image:: https://img.shields.io/pypi/v/pygpcca
+    :target: https://pypi.org/project/pygpcca/
+    :alt: PyPI
+
+.. |Conda| image:: https://img.shields.io/conda/vn/conda-forge/pygpcca
+    :target: https://anaconda.org/conda-forge/pygpcca
+    :alt: Conda
+
+.. |Cite| image:: https://img.shields.io/badge/DOI-10.1021%2Facs.jctc.8b00079-blue
+    :target: https://doi.org/10.1021/acs.jctc.8b00079
+    :alt: Cite
+
+.. |CI| image:: https://img.shields.io/github/workflow/status/msmdev/pygpcca/CI/main
+    :target: https://github.com/msmdev/pygpcca/actions
+    :alt: CI
+
+.. |Docs|  image:: https://img.shields.io/readthedocs/pygpcca
+    :target: https://pygpcca.readthedocs.io/en/latest
+    :alt: Documentation
+
+.. |Coverage| image:: https://img.shields.io/codecov/c/github/msmdev/pygpcca/main
+    :target: https://codecov.io/gh/msmdev/pygpcca
+    :alt: Coverage
+
+.. |License| image:: https://img.shields.io/github/license/msmdev/pyGPCCA?color=green
+    :target: https://github.com/msmdev/pyGPCCA/blob/main/LICENSE.txt
+    :alt: GitHub
+
+.. |GitHubDownloads| image:: https://img.shields.io/github/downloads/msmdev/pyGPCCA/total?label=github%20downloads
+    :target: https://github.com/msmdev/pyGPCCA/releases/
+    :alt: GitHub all releases
+
+.. |PyPIdownloads| image:: https://img.shields.io/pypi/dm/gpcca?label=pypi%20downloads
```

Comment:
This is already done above, see |PyPI|. If you want the number of downloads from PyPI, you can use: http://pepy.tech/

Reply:
Done.